test/e2e/framework/util.go

```diff
@@ -39,7 +42,7 @@ const (
 	PodListTimeout = time.Minute
 	// Initial pod start can be delayed O(minutes) by slow docker pulls
 	// TODO: Make this 30 seconds once #4566 is resolved.
-	PodStartTimeout = 5 * time.Minute
+	PodStartTimeout = 1 * time.Minute
```

Comment:
@m1093782566 Can you please set it back to ```5 * time.Minute``` as before.
On local clusters runs it passes with the timeout of one minute, but we need more time for our tests to run on test-infra clusters.

Because of one minute time-out the first test case itself is faliling, in the past two run.

Reply:
Okay, will try...